Angular InstantSearch/manipulating-lists/sorting/src/app/app.component.ts

```diff
@@ -1,5 +1,11 @@
 import { Component } from '@angular/core';
 import { sortBy } from 'lodash';
+import algoliasearch from 'algoliasearch/lite';
```

Comment:
quotes

Reply:
It is consistent with the file. We need to run Prettier on another PR.